modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -7517,6 +7517,7 @@ if-enabled-this-page-will-inherit-changes-made-to-the-page-template=If enabled,
 if-enabled-this-page-will-inherit-changes-made-to-the-x-page-template=If enabled, this page will inherit changes made to the <em>{0}</em> page template.
 if-everything-is-configured-correctly,-but-you-still-encounter-this-error,-the-administrator-has-the-option-to-forcibly-disable-remote-staging=If everything is configured correctly, but you still encounter this error, the administrator has the option to forcibly disable remote staging.
 if-the-discount-type-is-free-shipping,-shipping-charges-are-subtracted-from-the-order=If the discount type is <em>Free Shipping</em>, shipping charges are subtracted from the order. If the discount type is <em>Tax Free</em>, then tax charges are subtracted from the order.
+if-the-search-configuration-of-this-object-field-is-update=If the search configuration of this object field is update, go to Control Panel > Search > Index Actions and execute the reindex for the current object so the changes are applied.
```

Comment:
@GabrielHenriquedev Just a small adjustment. It should be "updated" instead of "update".